pkg/apiserver/topo/topo.go

```diff
@@ -0,0 +1,253 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// topo is a directory for TopoServer, which could load topology from pd
+// using Etcd v3 interface and pd interface.
+
+package topo
```

Comment:
Prefer to organize it in `clusterinfo` since we will add more (not only topology).